### ISSUE TITLE
Script to clean up validation and test set - to remove unseen entities/relations during train

### ISSUE THREAD

**Issue by sumitpai** [Accenture/AmpliGraph#62]
**Background and Context**
If the test/validation sets have entities that were not present during training, the ampligraph crashes with an error. 

**Description**
1. We need to have a graceful exit(raise an exception with meaningful message)

2. We need to have a script to help the user clean up test/valid set of entities/relations unseen during train. (Move clean_data from predictive_performance.py to a meaningful location and update documentation. )

3. Remove unseen entities while loading fb15k-237 and wnrr18

**Comment by sumitpai** [comment]
@chanlevan Can you look at points 2(documentation) and 3
